### PR TITLE
📦 Release v0.5.1

### PR DESCRIPTION
**Pull request by aslakhellesoy**
See diff for details.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (aurelien-reeves) [inline CHANGELOG.md]:
```suggestion
## [0.5.1]
```

Comment (aslakhellesoy) [reply CHANGELOG.md]: We either need to change the manual instructions to make this update manually, or adopt the `update-changelog` script we have in some of the other repos.

Comment (aurelien-reeves) [reply CHANGELOG.md]: Manual instructions are up-to-date:
```markdown
4. Update the CHANGELOG.md file, adding your changes in a section **beneath the new release number**.
```

Comment (aslakhellesoy) [reply CHANGELOG.md]: Hahaha I missed it 😅

Comment (ciaranmcnulty) [reply CHANGELOG.md]: Maybe we need a step 'add a new release number if one is needed' for clarity

**Pull request merged**